apps/nextra/pages/en/build/move/examples/first-move-module.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: "Your First Move Module"
+---
+
```

Comment:
I thought these were removed?

Reply:
We don't have a single guide on how to write move for devs in the docs. I feel like we need at least one.

Reply:
Okay, then let's get this in... and then we'll have to update it.